contributing.md

```diff
@@ -6,7 +6,13 @@ Start a new [Discussion](https://github.com/tendermint/starport/discussions/new)
 
 Thank you for your contribution!
 
-## Opening pull requests 
+## Feedback / Issues
+
+* Search through Google and [existing open and closed GitHub Issues](https://github.com/tendermint/starport/issues) for the answer first. If you find a relevant topic, please comment on the issue.
+* If none of the issues are relevant, please add an issue to [GitHub issues](https://github.com/tendermint/starport/issues/new/choose). Do provide any relevant information, case study or informative links as suggested by the Pull Request template.
```

Comment:
```suggestion
* To provide feedback or ask a question, create a [GitHub issue](https://github.com/tendermint/starport/issues/new/choose). Be sure to provide the relevant information, case study, or informative links as suggested by the Pull Request template. 
```

Reply:
@soniasingla I applaud your helpful tone, it's just right! Because we're friendly developers, we don't need "please", instead we follow the [Google developer documentation style guide](https://developers.google.com/style/tone#politeness-and-use-of-please).